lib/dictionaries/api.js

```diff
@@ -37,7 +37,16 @@ module.exports = function(opts){
   });
 
   router.delete('/:scope/:uuid/dictionaries/:name.json', parametrize, authorize, function(req, res, next) {
+    var filters = {'meta.uuid': req.params.uuid, 'meta.scope': req.params.scope, name: req.params.name};
+    var handleResponse = function(dictionary) {
```

Comment:
I would return the dictionary value instead of the wrapped dictionary. That's what you'd expect of a transaction